owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/logging/LogInterceptor.kt

```diff
@@ -85,7 +85,10 @@ class LogInterceptor : Interceptor {
     private fun logHeaders(headers: Headers): Map<String, String> {
         val auxHeaders = headers.toMap().toMutableMap()
         if (auxHeaders.contains(AUTHORIZATION_HEADER)) {
-            auxHeaders[AUTHORIZATION_HEADER] = "[redacted]"
+            val authHeaderList = auxHeaders[AUTHORIZATION_HEADER]!!.split(" ")
+            val authType = authHeaderList[0]
+            val authInfo = if (redactAuthHeader) "[redacted]" else authHeaderList[1]
```

Comment:
The same that before. I would write the string in the companion object.

Reply:
This is the same string as before, so the answer is the same